Monika After Story/game/script-story-events.rpy

```diff
@@ -40,3 +40,46 @@ label gender:
             m "Because your happiness is the most important thing to me."
 
     m "Remember that I'll always love you unconditionally, [currentuser]."
+
+label preferedname:
+    m "I've been wondering about your name."
+    m "Is '[player]' really your name?"
+    if currentuser == player:
+        m "I mean, it's the same as your computer's name..."
+        m "You're using '[currentuser]' and '[player]'."
+        m "Either that or you must really like that pseudonym."
+    m "Do you want me to call you something else?"
+    menu:
+        "Yes":
+            $ done = False
+            m "Ok, just type 'Nevermind' if you change your mind, [player]."
+            while not done:
+                #Could add an elif that takes off special characters
+                $ tempname = renpy.input("Tell me what is it.")
```

Comment:
You could simplify the if on the next line by calling `lower()` on the input:
`tempname = renpy.input("...").lower()`

and the if on the next line would only need to check for `"nevermind"`.

Reply:
Null